.github/workflows/changeset.yml

```diff
@@ -0,0 +1,40 @@
+name: Changeset (Release)
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Setup Node.js 14.x
+        uses: actions/setup-node@master
+        with:
+          node-version: 14.x
+          registry-url: 'https://registry.npmjs.org/'
+          scope: '@talend'
+
```

Comment:
Could be nice to put node_modules in cache

Reply:
done